python/ray/tune/tune.py

```diff
@@ -762,7 +791,7 @@ class and registered trainables.
         for exp in experiments:
             search_alg.add_configurations([exp])
     else:
-        logger.info(
+        logger.debug(
             "TrialRunner resumed, ignoring new add_experiment but "
             "updating trial resources."
         )
```

Comment:
What is this talking about?

Reply:
Any new experiments/configurations passed to `tune.run` will be ignored (we only continue the current state). This is when people pass `tune.run(different_experiment)` when resuming.

However, overwriting trainables is now a default way

Can we maybe
- Detect if an Experiment was passed or just a trainable (in this block `if not isinstance(exp, Experiment)`)
- If an experiment, continue to use the INFO message (maybe with updated wording)
- Else, don't print anything

Alternatively, we can keep it as is in the PR. I don't think anybody really passes experiments anyway and the message was unhelpful to begin with.

Reply:
I've kept it as a DEBUG, and improved the message a bit. It was a a bit more complicated to tell if the user passed in an experiment, since all trainables get converted to experiment -- felt that keeping it at the DEBUG log level was good enough.